mmdet/evaluation/metrics/coco_metric.py

```diff
@@ -73,7 +73,8 @@ def __init__(self,
                  outfile_prefix: Optional[str] = None,
                  file_client_args: dict = dict(backend='disk'),
                  collect_device: str = 'cpu',
-                 prefix: Optional[str] = None) -> None:
+                 prefix: Optional[str] = None,
+                 cat_ids: List[int] = None) -> None:
```

Comment:
unnecessary arguments.

Reply:
https://github.com/open-mmlab/mmdetection/blob/5b0d5b40d5c6cfda906db7464ca22cbd4396728a/mmdet/evaluation/metrics/coco_metric.py#L387-L392

As you can see,we should give the port to define cat_ids by ourselves.